src/indexers/citadel-indexer.ts

```diff
@@ -149,6 +150,9 @@ export async function snapshotCitadelMetrics() {
   const stakingApr = ((citadelMintedToStaking * ONE_YEAR_SECONDS) / duration.toNumber() / staked) * 100;
   citadelData.set('stakingApr', stakingApr);
 
+  const lockingAprs = await getRewardsAprForDataBlob();
+  citadelData.set('lockingApr', JSON.stringify(lockingAprs));
```

Comment:
i think if you use get property and set the type specifically in the blob you should get a proper serialization but i'm not sure since tbh i haven't tried. if it don't work then yeah this is the way and we should probably build this abstraction layer into the data blob itself (future task)